aptos-move/move-examples/tic-tac-toe/sources/TicTacToe.move

```diff
@@ -0,0 +1,639 @@
+module tic_tac_toe::ttt {
+    use std::vector;
+    use std::option::{Self, Option};
+    use std::signer;
+    use std::error;
+    use aptos_framework::account;
+    use aptos_framework::event;
+
+    #[test_only]
+    use aptos_framework::account::create_account_for_test;
+
+
+
+    //// GAME CONSTANTS
+    const GRID_SIZE: u64 = 9;
+    const WIDTH_AND_HEIGHT: u64 = 3;
+    const EMPTY_CELL: u64 = 3;
+
+    //// PLAYER CONSTANTS
+    const PLAYER_X_TYPE: u64 = 1;
+    const PLAYER_O_TYPE: u64 = 2;
+
+    //// ERROR CODES
+    const ERR_ILLEGAL_ACTION: u64 = 0;
+    const ERR_INVALID_MOVE: u64 = 1;
+    const ERR_PLAYER_TAKEN: u64 = 2;
+    const ERR_GAME_NOT_DONE: u64 = 3;
+
+    struct GameOverEvent has drop, store {
+        is_game_over: bool,
+    }
+
+    struct Player has copy, drop, store {
+        type: u64,
+        owner: address,
+    }
+
+    struct Board has drop, store {
+        vec: vector<u64>,
+        row: u64,
+        col: u64,
+    }
+
+    struct Game has key, store {
+        board: Board,
+        player_x: Option<Player>,
+        player_o: Option<Player>,
+        is_player_x_turn: bool,
+        is_game_over: bool,
+        game_over_events: event::EventHandle<GameOverEvent>,
+    }
```

Comment:
I wonder can you make this droppable and then you don't need to have the cleanup method?

Reply:
I had issues with `Game` being droppable because supposedly `event::EventHandle<GameOverEvent` wasn't droppable. I also explored the option of creating a separate struct to maintain `GameEvents` but I figured it was easier to keep it within Game and destroy the object at the end/forfeit

Reply:
Ah yes, I forgot about that I had similar issues.  I think the events even if they're droppable the handle isn't